lrlex/src/lib/lexer.rs

```diff
@@ -252,7 +365,7 @@ where
         Self: Sized;
 
     /// Instantiate a lexer from a string (e.g. representing a `.l` file).
-    fn from_str(s: &str) -> LexBuildResult<Self>
+    fn from_str(s: &str) -> Result<Self, Vec<LexBuildError>>
```

Comment:
Unnecessary change.

Reply:
Should be fixed in [223d257](https://github.com/softdevteam/grmtools/pull/556/commits/223d257af1349f8a3f49fb7ebcde2a3b0462c5a1)